spec/job_spec.rb

```diff
@@ -37,6 +37,21 @@
     end
   end
 
+  context '#job_plays' do
+    let(:url)            { "example.com/api/v1/job_plays" }
+    let(:job_collection) { build(:response_collection, :klass => described_class) }
+    let(:job_events)     { build(:response_url_collection, :klass => AnsibleTowerClient::JobEvent, :url => url) }
+    it "returns a collection of AnsibleTowerClient::JobEvents" do
```

Comment:
With the current discussion in regards to a `JobPlay` class for results off of the `/job_plays` endpoint - I suggest we merge this as is and I will build a new PR to address the type override.